app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.ProfileChooserViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.core.widget.NestedScrollView
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/profileChooserBackground"
+      android:overScrollMode="never"
+      android:scrollbars="none">
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/profile_chooser_container"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:gravity="end"
+          android:orientation="horizontal"
+          android:paddingTop="24dp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintTop_toTopOf="parent">
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:minHeight="24dp"
+            android:text="@string/profile_chooser_language"
+            android:textColor="@color/profileChooserGreyTextColor"
+            android:textSize="16sp" />
+
+          <ImageView
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="20dp"
+            android:src="@drawable/ic_settings_grey_48dp" />
```

Comment:
Need to replace with world 001 image. @mschanteltc please upload image in google drive.

Reply:
I uploaded the pictures in the Google Drive under Profile Chooser > Mock Profile Pictures

Reply:
@nikitamarysolomanpvt You can update this icon now.

Reply:
> I uploaded the pictures in the Google Drive under Profile Chooser > Mock Profile Pictures

@mschanteltc I need world 001 image.. Please check the mock https://xd.adobe.com/spec/e2239cf4-9cde-4c08-5296-25316c1f0a14-9412/screen/18e69208-8528-424a-a368-b2c5d530d235/PC-SP-Profile-Chooser-

Reply:
My mistake, I added the language icon under "Profile Chooser."